src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.props

```diff
@@ -12,6 +12,5 @@ Copyright (c) .NET Foundation. All rights reserved.
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackTool</TargetsForTfmSpecificContentInPackage>
```

Comment:
Since I added a `PackAsTool` condition to the PackTool.targets imports in Sdk.Common.targets, we can't unconditionally add this anymore. We don't know if `PackAsTool` has been set at this point (because we're pre-project-file-contents in the MSBuild evaluation ordering), so this should get pushed to the PackTool.targets instead. It doesn't effect ordering in anyway, so this is safe.